### PR TITLE
add docs on init hooks

### PR DESCRIPTION
**Pull request by HarshCasper**

### CODE REVIEW

File: content/en/references/init-hooks/index.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+title: "Initialization Hooks"
```
Comment (hovaesco) [inline content/en/references/init-hooks/index.md:2]: Shouldn't this be under user guides?

File: content/en/references/init-hooks/index.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+linkTitle: "Initialization Hooks"
+weight: 5
+description: >
+  Writing QQL scripts to initialize your Snowflake emulator
```
Comment (hovaesco) [inline content/en/references/init-hooks/index.md:6]:
```suggestion
  Writing SQL scripts to initialize resources in your Snowflake emulator
```

**Review by hovaesco — COMMENTED** [review]
Looks good!! Thanks for quickly jumping on it!

File: content/en/references/init-hooks/index.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+description: >
+  Writing SQL scripts to initialize resources in your Snowflake emulator
+cascade:
+  type: docs
+hide_readingtime: true
+---
+ 
+## Introduction
+
+LocalStack for Snowflake supports automatically executing `*.sf.sql` with [Init Hooks](https://docs.localstack.cloud/references/init-hooks/) when mounted into the Docker container. A script can be added to one of these stages in the lifecycle:
```
Comment (whummer) [inline content/en/references/init-hooks/index.md:14]:
```suggestion
LocalStack for Snowflake supports automatically executing `*.sf.sql` files via [Init Hooks](https://docs.localstack.cloud/references/init-hooks/) when mounted into the Docker container. A script can be added to one of these stages in the lifecycle:
```

**Review by whummer — APPROVED** [review]
Kudos for jumping on this @HarshCasper ! 🚤  LGTM, modulo the comment from @hovaesco

**Pull request merged**